libraries/standard/mqtt/src/mqtt_lightweight.c

```diff
@@ -20,8 +20,10 @@
  */
 
 #include <string.h>
+#include <assert.h>
 
 #include "mqtt_lightweight.h"
+#include "private/mqtt_internal.h"
```

Comment:
I do not like the name of this file, it is not an internal secret and it is not internal private functions. It is the logging interface, can we not just call it that?

Reply:
And I do not see a good reason for this to be "private" in the first place ...

Reply:
The inclusion of logging has not yet been finalized; this file was added to unblock ongoing development. If we later decide to change the file name, we can just change the include